readme.md

```diff
@@ -61,25 +61,34 @@ docker-compose up
 
 start service
 ```shell
-yarn service
+yarn service start
 ```
 
 start storybook
 ```shell
-yarn storybook
+yarn client dev
```

Comment:
my workflows here have moved more towards docker I think. What are you using?

Reply:
@chewnoill been using the package.json script commands to run the app out of habit - I'll delete the startup instructions besides docker

Reply:
yarn scripts are probably fine, maybe less overhead? If we can get the nextjs middleware working for websockets it would be viable.